src/translations/en.json

```
@@ -750,6 +750,7 @@
   "SavedCardDetails.replaceCardTitle": "Use another payment card",
   "SavedCardDetails.savedPaymentMethodPlaceholderDesktop": "•••• •••• •••• {last4Digits}",
   "SavedCardDetails.savedPaymentMethodPlaceholderMobile": "…{last4Digits}",
+  "SearchCTA.ButtonLabel": "Search",
```

```suggestion
  "SearchCTA.buttonLabel": "Search",
```

or   "SearchCTA.submitButtonLabel": "Search",